package.json

```diff
@@ -14,8 +14,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.4",
-    "@docusaurus/preset-classic": "2.0.0-beta.4",
+    "@docusaurus/core": "^2.0.0-beta.8",
+    "@docusaurus/preset-classic": "^2.0.0-beta.8",
```

Comment:
Jean also updated this in his PR:

https://github.com/lukso-network/docs/pull/33

Last one to merge may have conflicts 😁😅

Reply:
👍

Reply:
And Reto, you are the merge conflict winner haha 🙈

Reply:
![image](https://user-images.githubusercontent.com/798709/139838459-e9f117eb-412f-4e5c-8248-3af2a5f98104.png)
😄